tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/resources/eml/calendar_reply_declined-fr.eml

```diff
@@ -0,0 +1,5 @@
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Subject: Diminué:
```

Comment:
```suggestion
Subject: Rejeté:
```

Reply:
The mime message after parsing it is `DÃ©clinÃ©` 
is it normal?